documentation/docs/reference/services/Event.md

```diff
@@ -411,12 +411,12 @@ Request should include an attendee's user token (`userToken`) and an event id (`
 
 Returns a status, the user's new points, and the user's total points.
 
-Valid values for `status` are `Success`, `ExpiredOrProspective`, `AlreadyCheckedIn`. 
+Valid values for `status` are `Success`, `InvalidEventId`, `ExpiredOrProspective`, `AlreadyCheckedIn`. 
 
 !!! warning
 	Please be aware that `ExpiredOrProspective` will be returned if the user token is invalid, expired, or malformed. 
```

Comment:
In hindsight, a clearer error might be to just say `BadToken` for this generic definition. I wouldn't say it is crucial to have separate errors for when we receive an expired token or a malformed token since the solution for all of those outcomes is to just poll `GET /user/qr/`.

Reply:
Agreed. Since we're having a separate `InvalidEventId` and `InvalidCode` anyways, they don't need to match.

I added `BadUserToken` instead of `BadToken` (to be consistent with `InvalidEventId`), really was just a simple one-line change, should be good to go!

Resolved by 525a433.